.github/workflows/release-internal.yml

```diff
@@ -8,7 +8,7 @@
 #
 # When?
 #
-# Manual trigger.
+# Manual trigger or cron job every wednesday morning.
```

Comment:
Can we describe the fact this job doesn't do anything if there are no new commits